spec/spec_helper.rb

```diff
@@ -25,6 +25,18 @@ def create_user
            oauth_token: "xxx11")
 end
 
+def stub_bad_omniauth
+  OmniAuth.config.test_mode = true
+  OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+    info: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.